Index: .classpath
===================================================================
diff --git a/.classpath b/.classpath
deleted file mode 100644
--- a/.classpath	
+++ /dev/null	
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
Index: .project
===================================================================
diff --git a/.project b/.project
deleted file mode 100644
--- a/.project	
+++ /dev/null	
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
-	<name>LearnJava</name>
-	<comment></comment>
-	<projects>
-	</projects>
-	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-	</buildSpec>
-	<natures>
-		<nature>org.eclipse.jdt.core.javanature</nature>
-	</natures>
-</projectDescription>
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	
+++ /dev/null	
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/modules.xml	
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/Learn-Java.iml" filepath="$PROJECT_DIR$/Learn-Java.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: src/learnJava/Variables/StaticVariable.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/learnJava/Variables/StaticVariable.java b/src/learnJava/Variables/StaticVariable.java
new file mode 100644
--- /dev/null	
+++ b/src/learnJava/Variables/StaticVariable.java	
@@ -0,0 +1,8 @@
+package learnJava.Variables;
+
+public class StaticVariable {
+    static int num=2;
+    public static void main(String[] args){
+        System.out.println(num);
+    }
+}
Index: src/learnJava/Variables/InstanceVariable.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/learnJava/Variables/InstanceVariable.java b/src/learnJava/Variables/InstanceVariable.java
new file mode 100644
--- /dev/null	
+++ b/src/learnJava/Variables/InstanceVariable.java	
@@ -0,0 +1,9 @@
+package learnJava.Variables;
+
+public class InstanceVariable {
+    int instVar = 10;
+    public static void main(String[] args){
+        InstanceVariable instanceVariableObj = new InstanceVariable();
+        System.out.println(instanceVariableObj.instVar);
+    }
+}
Index: src/learnJava/Variables/LocalVariable.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/learnJava/Variables/LocalVariable.java b/src/learnJava/Variables/LocalVariable.java
new file mode 100644
--- /dev/null	
+++ b/src/learnJava/Variables/LocalVariable.java	
@@ -0,0 +1,17 @@
+package learnJava.Variables;
+
+import java.util.Scanner;
+
+public class LocalVariable {
+    public static void main(String[] args){
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter a number: ");
+        int num = sc.nextInt();
+        if(num %2 == 0){
+            System.out.println("Even number");
+        }
+        else{
+            System.out.println("Odd number");
+        }
+    }
+}
Index: src/learnJava/Variables/Variables.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/learnJava/Variables/Variables.java b/src/learnJava/Variables/Variables.java
new file mode 100644
--- /dev/null	
+++ b/src/learnJava/Variables/Variables.java	
@@ -0,0 +1,14 @@
+package learnJava.Variables;
+
+public class Variables {
+    int  num = 10; // non-static(instance) variable
+    static int statNum = 5; // static variable
+
+    public static void main(String[] args){
+        int num = 1; // local variable
+        System.out.println("Local variable: "+num);
+        System.out.println("Static variable: "+statNum);
+        Variables variablesObj = new Variables();
+        System.out.println("Instance variable: "+variablesObj.num);
+    }
+}
Index: Learn-Java.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Learn-Java.iml b/Learn-Java.iml
new file mode 100644
--- /dev/null	
+++ b/Learn-Java.iml	
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
